### PR TITLE
[CNFT2-2248] - Fix history pagination bug

### PR DESCRIPTION
**Pull request by lramakr**
## Description

Fixed history pagination bug
Screenshot 2024-03-26 at 12 34 56 PM

## Tickets

* [Jira Ticket] https://cdc-nbs.atlassian.net/browse/CNFT2-2248

## Checklist before requesting a review
- [x] PR focuses on a single story
- [x] Code has been fully tested to meet acceptance criteria
- [x] PR is reasonably small and reviewable (Generally less than 10 files and 500 changed lines)
- [x] All new functions/classes/components reasonably small
- [x] Functions/classes/components focused on one responsibility
- [x] Code easy to understand and modify (clarity over concise/clever)
- [x] PRs containing TypeScript follow the [Do's and Don'ts](https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)
- [x] PR does not contain hardcoded values (Uses constants)
- [x] All code is covered by unit or feature tests

### CODE REVIEW

File: apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx [hunk 1]
```diff
@@ -48,12 +48,15 @@ const PageInformation = () => {
     };
     useEffect(() => {
         fetchPageInfo();
-        fetchPageHistory();
+        fetchPageHistory().then();
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx:51]: Issue: `.then()` doesn't do anything

File: apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx [hunk 2]
```diff
@@ -48,12 +48,15 @@ const PageInformation = () => {
...
     }, [page]);
 
+    useEffect(() => {
+        fetchPageHistory().then();
```
Comment (mpeels) [inline apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx:55]: Issue: .then() doesn't do anything

**Review by mpeels — CHANGES_REQUESTED** [review]

File: apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx (hunk 1 above)
Comment (lramakr) [reply apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx:51]: That is just to satisfy the lint warning

File: apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx (hunk 2 above)
Comment (lramakr) [reply apps/modernization-ui/src/apps/page-builder/page/management/preview/information/PageInformation.tsx:55]: That is just to satisfy the lint warning

**Review by alaapbharadwaj — APPROVED** [review]

**Review by alaapbharadwaj — APPROVED** [review]

**Review by mpeels — APPROVED** [review]

**Pull request merged**